.CMake/alg_support.cmake

```
@@ -168,6 +168,15 @@ cmake_dependent_option(OQS_ENABLE_SIG_sphincs_shake_256f_simple "" ON "OQS_ENABL
 cmake_dependent_option(OQS_ENABLE_SIG_sphincs_shake_256s_simple "" ON "OQS_ENABLE_SIG_SPHINCS" OFF)
 ##### OQS_COPY_FROM_UPSTREAM_FRAGMENT_ADD_ENABLE_BY_ALG_END
 
+##### OQS_COPY_FROM_LIBJADE_FRAGMENT_ADD_ENABLE_BY_ALG_START
+if ((OQS_LIBJADE_BUILD STREQUAL ON))
```

I'm not sure it makes a difference or not, but it matches the conventions in the rest of the file.
```suggestion
if ((OQS_LIBJADE_BUILD STREQUAL "ON"))
```